react/features/base/i18n/functions.js

```diff
@@ -1,14 +1,37 @@
+// @flow
+
 import React from 'react';
 import { withTranslation } from 'react-i18next';
 
+import i18next from './i18next';
+
+/**
+ * Changes the main translation bundle.
+ *
+ * @param {string} language - The language e.g. 'en', 'fr'.
+ * @param {string} options.url - The url of the translation bundle.
+ * @param {Object} options.data - The actual bundle.
+ * @returns {void}
+ */
+export async function changeLanguageBundle(language: string, { url, data }: {url?: string, data?: Object}) {
+    let bundle = data;
```

Comment:
what's the point of initializing with `data`?

Reply:
seems like we never pass it anyway

Reply:
Removed it